cognite/neat/rules/models/rules.py

```diff
@@ -553,29 +551,31 @@ def set_class_name_if_none(cls, value, values):
 
     @field_validator("parent_class", mode="before")
     @skip_field_validator("validators_to_skip")
-    def to_list_if_comma(cls, value, values):
+    def to_list_of_entities(cls, value, values):
         if isinstance(value, str):
             if value:
-                return value.replace(", ", ",").split(",")
+                parent_classes = []
+                for v in value.replace(", ", ",").split(","):
+                    try:
+                        parent_classes.append(ParentClass.from_string(entity_string=v))
+                    # if all fails defaults "neat" object which ends up being updated to proper
+                    # prefix and version upon completion of Rules validation
+                    except ValueError:
+                        parent_classes.append(ParentClass(prefix="undefined", suffix=value, name=value))
+
+                return parent_classes
             if cls.model_fields[values.field_name].default is None:
                 return None
```

Comment:
This is now obsolete as if this `if` condition fails then if will also return None. 
```suggestion
```